Python/ceval.c

```diff
@@ -408,6 +415,16 @@ Py_MakePendingCalls(void)
     if (busy)
         return 0;
```

Comment:
Would you mind to add assert(PyGILState_Check()); at the function entry to make it more explicit that this function requires to hold the GIL?

(Right now, it's not obvious for me if the pending_lock initialization is safe.)